docs/core/build/distribution-packaging.md

```diff
@@ -107,7 +107,7 @@ The following lists the recommended packages:
   - **Version:** \<runtime version>
   - **Example:** dotnet-sdk-2.1
   - **Contains:** (3),(4)
-  - **Dependencies:** `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
+  - **Dependencies:** `dotnet-runtime-[major].[minor]`, `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
```

Comment:
I just noticed that this line says `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]` but the list of packages below says `netstandard-targeting-pack-[major].[minor]`. That seems like a mismatch. It should be the netstandard version in both cases, right?

Reply:
I made it explicit here to make clear this is not matching the `[major].[minor]` of the package. I'll use it everywhere so it's less confusing.